configs/audio_recognition/README.md

```diff
@@ -0,0 +1,55 @@
+# Audio Recognition
+
+## Introduction
+
+## Model Zoo
+
+### Kinetics-400
+
+|config | n_fft | gpus | backbone |pretrain| top1 acc| top5 acc | inference_time(video/s) | gpu_mem(M)| ckpt | log| json|
+|:--|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|:--:|
+|[tsn_r18_64x1x1_100e_kinetics400_audio_feature](/configs/audio_recognition/tsn_r50_64x1x1_100e_kinetics400_audio_feature.py)|1024|8| ResNet18 | None |19.7|35.75|x|1897|[ckpt](https://download.openmmlab.com/mmaction/recognition/audio_recognition/tsn_r18_64x1x1_100e_kinetics400_audio_feature/tsn_r18_64x1x1_100e_kinetics400_audio_feature_20201012-bf34df6c.pth)|[log](https://download.openmmlab.com/mmaction/recognition/audio_recognition/tsn_r18_64x1x1_100e_kinetics400_audio_feature/20201010_144630.log)|[json](https://download.openmmlab.com/mmaction/recognition/audio_recognition/tsn_r18_64x1x1_100e_kinetics400_audio_feature/20201010_144630.log.json)|
+|[tsn_r50_64x1x1_100e_kinetics400_audio_feature + tsn_r50_1x1x3_100e_kinetics400_rgb]()|1024|8| ResNet(18+50) | None |70.01|88.71|x|x|x|x|x|
```

Comment:
Does this setting need a config link ?

Reply:
Yeah, I forgot. Two links for two configs.